tests/query_rules/10_query_rules.yml

```diff
@@ -0,0 +1,49 @@
+---
+requires:
+  serverless: true
+  stack: true
+---
+setup:
+  - do:
+      query_rules.put_ruleset:
```

Comment:
Should this be in `20_rulesets.yaml`?

Reply:
I created a ruleset here since as a dependency for query rules, but we're testing it in 20_rulesets too.

Reply:
Ah right this is the setup phase, sorry. Thanks, LGTM!